Branch-TestBed/Branch-TestBed.xcodeproj/project.pbxproj

```diff
@@ -208,7 +203,7 @@
 				ORGANIZATIONNAME = "Branch Metrics";
 				TargetAttributes = {
 					6700165F1940F51400A9E103 = {
-						DevelopmentTeam = 6B2DFWM843;
```

Comment:
If you guys don't have one specific team that you use, then it's not a big deal. At that point, just a churn things. I usually try not to commit things that are not related to the pull request. In this case, the team isn't related to anything, and probably shouldn't be committed at all, or could be committed as "None" to prevent this churn.